.travis.yml

```diff
@@ -1,4 +1,6 @@
 language: generic
+# Use Ubuntu 16.04
+dist: xenial
```

Comment:
Should this be specifically in the `-os: linux` sections?

Reply:
Putting this here would apply to all linux jobs. Otherwise we'll have to add this for each of them.

Reply:
Ok, that's fine.